runbotics/runbotics-orchestrator-ui/src/pages/blog/index.tsx

```diff
@@ -45,22 +42,16 @@ const BlogPage: VFC<Props> = ({ posts, categories, tags, page, featuredPost }) =
 
 export default BlogPage;
 
-export const getServerSideProps: GetServerSideProps<Props> = async ({ query, res, locale }) => {
+export const getServerSideProps: GetServerSideProps<Props> = async ({ query, locale }) => {
+    const language = locale as Language;
+    const cache = getBlogMainCache(language) ?? await recreateCache(language);
+
     const metadata: MetadataTags = {
         title: 'RunBotics | Blog',
         description: 'RunBotics - Blog',
         image: BlogBg.src,
     };
 
-    let cache = getBlogMainCache(locale as Language);
-
-    if (!cache) {
-        cache = await recreateCache(locale as Language);
-
-    } else {
-        res.setHeader('X-Cache', 'HIT');
-    }
-
```

Comment:
why remove header?

Reply:
![image](https://github.com/runbotics/runbotics/assets/82586165/5ea8c12b-ddc0-4f7d-866b-7e80d636c28f)

Above code inside _app.tsx is executed each time user entered the page, so to remove code duplication, I decided to move it.